--- libsigrokdecode-0.3.0/Makefile.am.orig	2014-05-06 23:08:31.000000000 +0200
+++ libsigrokdecode-0.3.0/Makefile.am	2014-07-27 19:41:48.489712604 +0200
@@ -71,7 +71,7 @@
 tests_runtc_SOURCES = tests/runtc.c
 tests_runtc_CPPFLAGS = $(CPPFLAGS_PYTHON) $(LIBSIGROK_CFLAGS) \
 	-DDECODERS_DIR='"$(abs_top_builddir)/decoders"'
-tests_runtc_LDFLAGS = -L$(top_builddir) $(LIBSIGROK_LIBS) $(LDFLAGS_PYTHON) -lsigrokdecode
+tests_runtc_LDADD = $(top_builddir)/libsigrokdecode.la $(LIBSIGROK_LIBS) $(LDFLAGS_PYTHON)
 endif
 
 MAINTAINERCLEANFILES = ChangeLog
